src/circuits/registry.ts

```diff
@@ -6,7 +6,8 @@ import { Query } from '@lib/circuits/query';
 import { Resolvers } from '@lib/state/resolver';
 
 export type VerifyOpts = {
```

Comment:
I guess the interface here will be enough `export type VerifyOpts` -> `export interface VerifyOpts`

Reply:
Done